Portugues.json

```
@@ -0,0 +1,159 @@
+{
+    "status": {
+        "terminated": "\nAplicacao Finalizada.\nFeche a janela para sair do programa.",
+        "goes_online": "{channel} esta ONLINE, Mudando...",
+        "goes_offline": "{channel} ficou OFFLINE, Mudando...",
+        "claimed_drop": "Coletou o drop: {drop}",
+        "claimed_points": "Coletou o bonus: {points}",
+        "earned_points": "Ganhou este valor de pontos: {points}, total: {balance}",
+        "no_channel": "Sem canal disponivel para assisitir. Esperando por um canal ONLINE...",
+        "no_campaign": "Sem campanhas ativas no momento para minerar. Esperando por uma campanha ativa..."
+    },
+    "login": {
+		"chrome": {
+    "startup": "Abrindo Google Chrome",
+    "login_to_complete": "Por Favor complete o login pela pagina web manualmente pressionando o botão Login novamente.",
+    "no_token": "Seu Token de autorização Twitch não foi encontrado",
+    "closed_window": "O aplicativo Goggle Chrome foi fechado , portanto o login não foi efetuado com sucesso .",
+},
+"error_code": "Login error code: {error_code}",

```

Indentation looks a little bit messed up.

```suggestion
            "startup": "Abrindo Google Chrome",
            "login_to_complete": "Por Favor complete o login pela pagina web manualmente pressionando o botão Login novamente.",
            "no_token": "Seu Token de autorização Twitch não foi encontrado",
            "closed_window": "O aplicativo Goggle Chrome foi fechado , portanto o login não foi efetuado com sucesso ."
        },
        "error_code": "Login error code: {error_code}",
```